ui/src/main/webapp/src/app/components/form.component.ts

```diff
@@ -24,15 +24,17 @@ export class FormComponent {
     handleError(error:any) {
         this.errorMessages = [];
         if (!error) {
-            this.errorMessages.push("Server call failed!");
-        } else if (error.parameterViolations) {
+            this.errorMessages.push("Server call failed.");
+        }
+        else if (error.parameterViolations) {
             error.parameterViolations.forEach(violation => {
                 console.log("Violation: " + JSON.stringify(violation));
                 this.errorMessages.push(violation.message);
             });
-        } else {
+        }
+        else {
```

Comment:
But we have 99.99% of our TypeScript code following that style. 
I see your point of having different code style for Java and TypeScript. I'm not fan of that either, but unfortunately best practices for Java and TypeScript differ here. (Sometimes they differ even for different libraries, which is quite odd :( ) And I'm for following best practices recommended for given language. We might get some contributions from someone who is purely TypeScript developer and expects that code style. 

Anyway, we could discuss that again later, but please, for now, keep it as it is and do not make unnecessary inconsistencies.

I'd love to add some code style rules and checker, but currently there are too many changes going on to do that...